packages/server/src/ChunkExtractor.js

```diff
@@ -248,11 +253,12 @@ class ChunkExtractor {
 
   getRequiredChunksScriptTag(extraProps) {
     return `<script id="${LOADABLE_REQUIRED_CHUNKS_KEY}" type="application/json"${extraPropsToString(
+      null,
       extraProps,
     )}>${this.getRequiredChunksScriptContent()}</script>`
   }
 
-  getRequiredChunksScriptElement(extraProps) {
+  getRequiredChunksScriptElement(asset, extraProps) {
```

Comment:
I don't understand this change. Why changing this signature, you can just call `handleExtraProps(null, extraProps)`.

Reply:
you are correct